pkgs/development/python-modules/mayavi/default.nix

```diff
@@ -18,8 +19,7 @@ buildPythonPackage rec {
   pname = "mayavi";
   version = "4.8.1";
   format = "setuptools";
-
-  disabled = pythonOlder "3.8";
+  disabled = pythonOlder "3.8" || pythonAtLeast "3.11";
```

Comment:
The PR, nor the commit message, nor the nix file itself includes a comment explaining why is this disabled. Note also https://github.com/NixOS/nixpkgs/commit/f409f152d226d488c402a9668fdbe41d8a47f4af which perhaps was merged in parallel without you knowing. I also wonder how is this relevant to the PR :)